packages/appkit/src/library/react/index.ts

```diff
@@ -23,11 +26,17 @@ type ThemeVariablesOptions = AppKitOptions['themeVariables']
 declare global {
   namespace JSX {
     interface IntrinsicElements {
+      'appkit-button': Pick<
+        AppKitButton,
+        'size' | 'label' | 'loadingLabel' | 'disabled' | 'balance'
+      >
+      'appkit-connect-button': Pick<AppKitConnectButton, 'size' | 'label' | 'loadingLabel'>
+      'appkit-account-button': Pick<AppKitAccountButton, 'disabled' | 'balance'>
+      'appkit-network-button': Pick<AppKitNetworkButton, 'disabled'>
       'w3m-connect-button': Pick<W3mConnectButton, 'size' | 'label' | 'loadingLabel'>
```

Comment:
For the newer versions we can drop the old component TS supports, this would caurage users to switch to `<appkit-* />` ones